src/napari/layers/_scalar_field/scalar_field.py

```diff
@@ -737,3 +671,95 @@ def _display_bounding_box_augmented_data_level(
         bounding box of the data at the current level
         """
         return self._extent_level_data_augmented[:, dims_displayed].T
+
+    def _get_layer_slicer(
+        self, data: LayerDataType, cache: bool
+    ) -> ScalarFieldSlicingState:
+        return ScalarFieldSlicingState(layer=self, data=data, cache=cache)
```

Comment:
Given the rename, I think we should replace all these `_layer_slicer` with `_slicing_state`, so we're fully distinct from the existing `_layer_slicer` on the viewer!